examples/paho-mqtt/Makefile

```diff
@@ -20,8 +20,8 @@ WIFI_PASS ?= "Your_secure_password"
 # Optionally include remoteDNS support. This includes resolution of names at an
 # upstream DNS server and the handling of RDNSS options in Router Advertisements
 # to auto-configure that upstream DNS server.
```

Comment:
It's now longer optionally if it is the default? I mean, not wrong but sounds misleading. I also believe you should add a pointer that disabling it is safe and yields smaller builds.